scripts/hab_conf_transfer_to_Q.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Transfer `habitat_confirmations.xls` data to `form_fiss_2023.gpkg`"
+date: "Created: 2024-04-26 | Updated: `r format(Sys.Date(), '%Y-%m-%d')`"
+output: 
+  html_document:
+    code_folding: "none"
+params:
+  repo_owner: "NewGraphEnvironment"
+  repo_name: "fish_passage_skeena_2023_reporting"
+---
+
+```{r setup, echo=TRUE, include = TRUE}
+knitr::opts_chunk$set(echo=TRUE, message=FALSE, warning=FALSE, dpi=60, out.width = "100%", eval = FALSE)
+options(scipen=999)
+options(knitr.kable.NA = '--') 
+options(knitr.kable.NAN = '--')
+```
+
+
+## Purpose
+
+This workflow is unique to the 2023 fiss form data and only needs to be done once. While populating the `habitat_confirmations.xls` spreadsheet, edits were made by hand to the `temperature`, `conductivity`, and `turbidity` columns in the `step_2_fish_coll_data` sheet. While doing so we realized we should QA `form_fiss_2023` in Q before populating the `habitat_confirmations.xls` spreadsheet so that we shouldn't need to make edits by hand, like in this case. 
+
+But we learn from our mistakes and will now go and QA `form_fiss_2023`. We don't need to duplicate our edits, so we will copy the edits made by hand in `habitat_confirmations.xls` to `form_fiss_2023.gpkg` before starting the QA. The purpose of this script is to add data from the `habitat_confirmations.xls` spreadsheet to the `form_fiss_2023.gpkg`. `eval = FALSE` because we only need to run this code once, and don't want to run this chunk when knitting the document.
+
+Moving forward we should use the following workflow:
+
+  1. QA `form_fiss_2023` in Q.
+  2. Run `fiss_site_tidy.R` to populate step 1 and step 4 of the `habitat_confirmations.xls` spreadsheet.
+  3. Run `extract_inputs.R` to populate step 2 of the `habitat_confirmations.xls` spreadsheet.
```

Comment:
hmm.  we should get the path and name correct when we document.  this file `name` does't exist in the main `scripts` directory (or anywhere else for that matter)

Reply:
will fix these thanks